### PR TITLE
Adds a Pride Week themed bar sign.

### PR DESCRIPTION
**Pull request by genessee596**
## About The Pull Request

image

Does what it says on the tin. Adds a new barsign to celebrate pride week, that's it. Doesn't do anything with other barsigns, and is totally optional for the bartender to set it to or whatever. Called 'The Pride Bar'.

## Why It's Good For The Game

A little nod to Pride Week for those who desire it, but nothing too flashy to avoid taking away from the main SS13 holiday events.

## Changelog
:cl:
add: Added a new bar sign to celebrate Pride Week.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/barsigns.dm [hunk 1]
```diff
@@ -289,6 +289,14 @@
...
 	desc = "Drink till you puke and/or break the laws of reality!"
 
+/datum/barsign/pridebar
+	name = "The Pride Bar"
+	icon = "pridebar"
+	desc = "Show your true colors and have a drink!"
+
+
+
+
```
Comment (bobbahbrown) [inline code/game/objects/structures/barsigns.dm:299]: why new lines
```suggestion
```

**Review by bobbah — CHANGES_REQUESTED** [review]

**Comment by bobbahbrown** [comment]
is this intended to be something that is testmerged for the duration of pride week? otherwise it seems like an odd addition to the codebase for the majority of the year

**Comment by genessee596** [comment]
> is this intended to be something that is testmerged for the duration of pride week? otherwise it seems like an odd addition to the codebase for the majority of the year

Testmerged for prideweek, yeah.

**Comment by MrPerson** [comment]
Gay bars are a thing all year round and it's nice to have a little nod to them that's not cringey.

**Comment by genessee596** [comment]
Fixing colors really quickly, right now it's kind of hard to read the text - please hold



> Gay bars are a thing all year round and it's nice to have a little nod to them that's not cringey.
> 
> EDIT: You could also hide it behind a holiday datum if you absolutely insist on it.

Meh, I'm lazy and stupid, so I'll avoid putting the holiday datum for now

File: code/game/objects/structures/barsigns.dm (hunk 1 above)
Comment (genessee596) [reply code/game/objects/structures/barsigns.dm:299]: Whoopsie. My bad!

**Comment by genessee596** [comment]
Edited sign text coloring to make it easier to read.

**Comment by ishitbyabullet** [comment]
Can you give the text outlines or something? It's hard to read.

**Comment by genessee596** [comment]
> Can you give the text outlines or something? It's hard to read.

I don't think there's enough room to do an outline? I'll double-check in a few hours, but I think it'll make it look crappy.

**Comment by wesoda25** [comment]
I think more frames would definitely help this one, seems very bare bones currently. Thinking stuff like flashing colors or the rainbow having some sort of movement.

**Comment by genessee596** [comment]
> I think more frames would definitely help this one, seems very bare bones currently. Thinking stuff like flashing colors or the rainbow having some sort of movement.

Remember, this is only getting testmerged for a week.

**Comment by carshalash** [comment]
Colons are a little hard to see and some of the choice of colors for the letters make it a lil hard to see.

**Comment by necromanceranne** [comment]
I'd make the colours bold dark colours to stand out from the already colourful background. Like, straight up a black would be fine.

Also

LET'S START A WAR
LET'S START A NUCLEAR WAR
AT THE GAY BAR, GAY BAR, GAY BAR

(if the nukie bar doesn't come set default to the pride flag I'll be deeply upset is what I'm saying)

**Comment by PKPenguin321** [comment]
Maybe it's because it's a pixel based sprite but I'm getting mad nyan cat vibes off of this

**Review by Twaticus — CHANGES_REQUESTED** [review]
Way too many clashing colors making it barely legible.

If youre going for a neon lights display, the backround needs to be muted with the text bright. Could be a black backround with rainbow text, or dark rainbow backround with light rainbow text. See thegreytide, maltesefalcon, theouterspess.

An LCD type display could really be anything as long as its readable. It could be as complicated as thenet or as simple as thewretchedhive. Maybe something like thesingulo where its a simple text but animated rainbow on the side?

The other option is something like alohasnackbar where it can straight up be a pride flag on the wall.

Sorry for wall of text but this sprite aint it chief hope this helps.

**Comment by TheVekter** [comment]
We're doing a bit of a bigger pride week thing and Genessee gave me permission to use this in that.

**Comment by genessee596** [comment]
> We're doing a bit of a bigger pride week thing and Genessee gave me permission to use this in that.

This is true, this has been rolled in with that.

**Comment by interestingusernam3** [comment]
>Colons are a little hard to see

Clearly the owners of those colons are not gay enough

**Pull request closed without merging**